### ISSUE TITLE
Bank filter additions

### ISSUE THREAD

**Issue by flyingjayjay** [oldschoolgg/oldschoolbot#5527]
Hello bso coders and readers,

So i have been collecting gems and dhide and few other things. I have not been the only one who has said adding a filter for key collection items for trading/seeing said items to be easier. (mainly a QOL)

So far ive seen: Ores, dhide/dleather, gems uncut/cuts, maybe magic runes

**Comment by DarkBugz** [comment]
Theres a skilling filter

**Comment by flyingjayjay** [comment]
> Theres a skilling filter

Yes theres a skilling filter but not specific to said items. if u traded said skilling filter it would be more than just gems etc.

**Comment by nwjgit** [comment]
There are already tons of filters and I don't think adding more will do any good. However I think a better idea would be to add user customizable filters. Similar to gear presets where we get a set amount of filters we can create and name. It would look similar to when how you view and add favorite alchs.

**Comment by TURB042O** [comment]
> There are already tons of filters and I don't think adding more will do any good. However I think a better idea would be to add user customizable filters. Similar to gear presets where we get a set amount of filters we can create and name. It would look similar to how you view and add favorite alchs. 

put that in as a feature request. or if you'd like and if you can't post in vote channels I can make a vote for you. you just type it out

**Comment by TURB042O** [comment]
btw there is

* `junk` - just runes
* `sm` - smithing (as in ores & bars)
* `fletch` - fletching
* `cr` - crafting 
* `seconds` - herblore seconds
* `herbs` - herbs
* `pots` - pots
